examples/send-plugin-message/pluginMessageTransfer.brs

```diff
@@ -0,0 +1,90 @@
+Function pluginMessage_Initialize(msgPort As Object, userVariables As Object, bsp as Object)
```

Comment:
A header comment, and maybe another comment on the _*_ProcessEvent(...)_ function, may help provide details about some of the hard requirements for plugin structures. 

See for more info, (private project ) https://github.com/brightsign/general-tools/blob/main/brightscript/plugins/plugin_template.brs